parts/masterparams.t

```diff
@@ -40,6 +40,13 @@
       "type": "string"
     },
     {{end}}
+    "masterSubnet": {
```

Comment:
I think we also want to remove the if statement that only adds this param value in some cases